### ISSUE TITLE
Layout needs improvements

### ISSUE THREAD

**Issue by Purfview** [techmovie/easy-seed#57]
1) Make order of the sites to be case insensitive (so "bB" would be at top, not at the bottom).
2) "Quick search:"  string looks out off place, maybe move it above the sites.
3) Make "Batch transfer" button a bit smaller, more near the size of the site names.
4) Move "Covert to thumbnails" button in a new row below Douban stuff.
5) Make all white buttons to look more like a button, maybe some color and style would help.

Here how it look initially and after both middle buttons pressed:

![alt text](https://i.imgur.com/AEbZ5CW.png)

![alt text](https://i.imgur.com/2taX1I4.png)

What you think about dual language (check if script at my fork works for you)?

**Comment by techmovie** [comment]
@Purfview 1 will be fixed.
2 do you mean the Quick Search string is too long?
3 change to a text button maybe look great.
4. move button to a new row may waste more space.Make the button more smaller may be a good choice
5. Dose the color of the button make you confused？🤔 The button style is from UI component Element.
6. Many thanks for your translation. Looks perfect.
Your way is more efficiency. I'm considering create sth like laguage config json though. It can reduce more conditional judgment.

**Comment by Purfview** [comment]
7) Convert <image> tags to <img>.

2) I meant to make it more ergonomic, like "Transfer to", to be on the same line with the site names.
4) Or maybe change "Working.../Success!" messages to colors of the button like "Yellow/Green". Jumping buttons around is not a good design in general. :)
5) Not confusing, just buttons look bland, maybe some more visible border would help.
6) Separate config module would be the best approach, but I doubt that anyone will want to translate it to more languages.

**Comment by techmovie** [comment]
Got it. I'll fix them all in next version(maybe 1.2.7😝)
6. English is enough

**Comment by Purfview** [comment]
`7.` That rotating config button consumes so much cpu/gpu for some reason. If  I apply my Darkmode script on top of it then it starts to consume a whole core.  :astonished:

**Comment by Purfview** [comment]
`8.`Rotating svg button sometimes doesn't work, like it doesn't register clicks.
On my fork I tried a png image and it solved the click problem, but cpu/gpu consumption is same, looks like that css animation eats a lot of cpu/gpu cycles on some styles. 
(Need to find some good gif creation program.)

**Comment by techmovie** [comment]
@Purfview Have you tested the cpu consumption is exactly caused by the rotate icon?I checked yesterday,didn't reproduce this issue.

**Comment by Purfview** [comment]
Yes, and I found info on internet that those css animations are cpu/gpu hungry, nothing can be done about them.

First I noticed unusual high speed on GPU fan, bit by bit I found that if you disable rotation on icon then GPU fan gets quiet, then I disabled hardware acceleration in browser and then CPU usage jumped up when I enabled rotation animation.

**Comment by techmovie** [comment]
@Purfview Maybe I should just remove the animation.The only purpose of the animation is drawing some attention in case of that users don't know where is setting panel.

**Comment by Purfview** [comment]
I like animated icon and yes it directs users where settings are.
Gif animation would solve cpu/gpu usage problem, but I couldn't find good gif on internet, need to DIY with some soft.

**Comment by techmovie** [comment]
@Purfview I'll try it out.

**Comment by Purfview** [comment]
Found that Photoshop has function to create animated gifs. I created one but problem is that small gifs(64x64) have visible artifacts which looks bad on a dark background. :anguished:
I think good solution would be an option in settings to turn off animation.
Check if png icon in my fork look good to you, I can PR it to solve randomly not registering mouse clicks on the svg icon.

**Comment by techmovie** [comment]
> Found that Photoshop has function to create animated gifs. I created one but problem is that small gifs(64x64) have visible artifacts which looks bad on a dark background. 😧
> I think good solution would be an option in settings to turn off animation.
> Check if png icon in my fork looks good to you, I can PR it to solve randomly not registering mouse clicks on the svg icon.

the cpu consumption maybe caused by this regexp: https://github.com/techmovie/easy-upload/commit/f3566d90d4085fb9a0356602334c89baa610e3b8#diff-c463069c12f49f9501c2ffedd601bef99a216ae6cc490fa93c1603baf0cccaa1R152.
You can test this torrent. https://passthepopcorn.me/torrents.php?id=242683&torrentid=920713

**Comment by Purfview** [comment]
No, it's caused by css "animation".
